sonic-xcvrd/xcvrd/sff_mgr.py

```diff
@@ -435,6 +436,11 @@ def task_worker(self):
                     # Skip if these essential routines are not available
                     continue
 
+                if isinstance(api, Sff8472Api):
+                    sfp.set_lpmode(False)
```

Comment:
@peterbailey-arista Can we check for the return value in both the cases and log error if it returns False?
Also, can you please help in fixing the built failure?